### ISSUE TITLE
Remove dependency on core-js

### ISSUE THREAD

**Issue by laino** [jimp-dev/jimp#769]
This is not something you should do for the user of your library, **ever**.

It's my job to provide your lib an environment it can run in, it's not the job of an image library to go messing with it.

Let me worry about polyfilling stuff or not polyfilling stuff. Now I have to ship core-js just to get jimp working.

Maybe have a ``@jimp/polyfills`` repo if you want to provide auto-polyfilling.

# Expected Behavior

No dependency on core-js.

# Current Behavior

Jimp crashes without core-js.

# Failure Information (for bugs)

```
Error: Cannot find module 'core-js/modules/es6.object.define-property'
Aug 06 10:58:11 sales hub[11767]: Require stack:
Aug 06 10:58:11 sales hub[11767]: - /var/hub/node_modules/jimp/dist/index.js
```

## Steps to Reproduce

Use jimp.

## Context

My application uses jimp.

- Jimp Version: 0.6.4
- Operating System: Any
- Node version: v12.7.0

## Failure Logs

```
Error: Cannot find module 'core-js/modules/es6.object.define-property'
Aug 06 10:58:11 sales hub[11767]: Require stack:
Aug 06 10:58:11 sales hub[11767]: - /var/hub/node_modules/jimp/dist/index.js
```

**Comment by pi0** [comment]
If you are using jimp **for Node.js** (not browser) you may try [jimp-compact](https://github.com/nuxt-community/jimp-compact) a repackaged version which is much smaller and also hasn't a dependency on jimp.

**NOTE:** This is **NOT** an official solution.

**Comment by hipstersmoothie** [comment]
@pi0 @laino PRs very welcome

**Comment by madconst** [comment]
As far as core-js now prints a lot of spam messages through its postinstall script being a typical donationware, it is now even more reasonable to remove dependency on it:
https://github.com/zloirock/core-js/issues/548

**Comment by l1bbcsg** [comment]
I can add that this ended up being a deal-breaker for me when selecting an image-processing utility for my commercial project.
Between spam, bloated size, questionable practice and "`added 62 packages from 93 contributors`" it's was simply not acceptable, even if jimp looked like a good choice, I'm quite disappointed in this myself.

**Comment by madconst** [comment]
> I can add that this ended up being a deal-breaker for me when selecting an image-processing utility for my commercial project.
> Between spam, bloated size, questionable practice and "`added 62 packages from 93 contributors`" it's was simply not acceptable, even if jimp looked like a good choice, I'm quite disappointed in this myself.

It's absolutely the same story in our project. We had to remove jimp for the same reason.

**Comment by EricRabil** [comment]
What if you guys literally just *removed the dependency on core-js*? It doesn't seem like that really needs an external contribution, given core-js just *polyfills the existing environment*. Literally all you need to do is run `npm remove core-js` in all of your packages... You guys really cba? This package has **1 million** weekly downloads, and it's disgusting that a leading library at that scale is still imposing core-js on its users in 2020.

I'll just start using the fork, and I'll fork the dependencies I have that rely on jimp to use the forked version as well. I'm not contributing to a library that can't be bothered to view maintaining the best developer experience as a top priority.

**Comment by hipstersmoothie** [comment]
pretty sure this was fixed with #1149
